Makefile

```diff
@@ -242,3 +242,9 @@ $(GTEST_OUT_DIR)/%_unittest.o: $(CXX_SRC_DIR)/query/%_unittest.cu
 
 $(GTEST_OUT_DIR)/all_unittest : $(patsubst %,$(GTEST_OUT_DIR)/%.o,$(CUDA_TESTS)) $(GTEST_OUT_DIR)/gtest_main.a
 	$(NVCC) $(CPPFLAGS) $(CXXFLAGS) $(NVCCFLAGS) $^ -o $@  -lpthread -lalgorithm
+
```

Comment:
there is no importable zookeep client lib?

Reply:
we are using `github.com/samuel/go-zookeeper`. do you mean a mock?